voxel51/users/auth.py

```diff
@@ -53,45 +53,81 @@ def deactivate_token():
     '''
     try:
         os.remove(TOKEN_PATH)
-        logger.info("Token '%s' successfully deactivated", TOKEN_PATH)
+        logger.info("API token '%s' successfully deactivated", TOKEN_PATH)
     except OSError:
-        logger.info("No token to deactivate")
+        logger.info("No API token to deactivate")
+
+
+def get_active_token_path():
+    '''Gets the path to the active API token.
+
+    If the ``VOXEL51_API_TOKEN`` environment variable is set, that path is
+    used. Otherwise, ``~/.voxel51/api-token.json`` is used.
+
+    Returns:
+        the path to the active token
+
+    Raises:
+        :class:`TokenError` if no token was found
+    '''
+    token_path = os.environ.get(TOKEN_ENVIRON_VAR, None)
+    if token_path is not None:
+        if not os.path.isfile(token_path):
+            raise TokenError(
+                "No API token found at '%s=%s'" %
+                (TOKEN_ENVIRON_VAR, token_path))
+    elif os.path.isfile(TOKEN_PATH):
+        token_path = TOKEN_PATH
+    else:
+        raise TokenError("No API token found")
```

Comment:
May be more user-friendly to say something like: "No API token for Voxel51 platform was found at /path/to/token. If you do not have a token, you can download one from https://console.voxel51.com/account/tokens"

Reply:
(I think there are a few places in this PR that could use an update like this)

Reply:
That already happens. Look at the definition of `TokenError`